setup.py

```diff
@@ -31,6 +31,7 @@
         'user-agents',
         'uwsgi',
         'werkzeug',
+        'yt',
```

Comment:
@robnagler is this a problem? I could work with the physicist on rsflash to make it so consumers of rsflash don't need to use yt.